docs/source/developer_guide/continuous_integration.rst

```diff
@@ -66,7 +66,8 @@ We recommend that you also set this up locally as it will ensure that you never
 These checks run automatically before you make a new commit.
 To setup, simply navigate to the sktime folder and install our pre-commit configuration:
 
-::
+.. code:: bash
+   
    pre-commit install
```

Comment:
```suggestion
   .. code:: bash

      pre-commit install
```

Code quality tests fail. Try something like this (not tested), and test locally with `pre-commit` please.

(I removed trailing spaces in the middle empty line, and added three spaces at start of both non-empty lines.)